src/js/msp/MSPHelper.js

```diff
@@ -237,6 +237,9 @@ MspHelper.prototype.process_data = function (dataHandler) {
                         FC.CONFIG.cpuTemp = data.readU16();
                     }
 
+                    if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_47)) {
```

Comment:
would a `if (data.byteLength > [...]` help here?

Reply:
i was just thinking not to bork in the wild 4.6 even though it is unofficial. i thought `if-statement` was an easy workaround.

Reply:
Yes confused about the value revealed as being 25

Reply:
This question is now resolved with latest firmware change as it keeps the number of rate profiles at 4 but now allows external define to set other values.